src/go-apps/blaze/go.sum

```diff
@@ -0,0 +1,143 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Can you tell me what this file is for? I literally hate it.

Reply:
[consistent dependencies](https://stackoverflow.com/questions/53837919/should-go-sum-file-be-checked-in-to-the-git-repository)